consul/consul.go

```diff
@@ -34,21 +35,21 @@ func (f *Factory) New(uri *url.URL) bridge.RegistryAdapter {
 	if uri.Scheme == "consul-unix" {
 		config.Address = strings.TrimPrefix(uri.String(), "consul-")
 	} else if uri.Scheme == "consul-tls" {
-	        tlsConfigDesc := &consulapi.TLSConfig {
-			  Address: uri.Host,
-			  CAFile: os.Getenv("CONSUL_CACERT"),
-  			  CertFile: os.Getenv("CONSUL_TLSCERT"),
-  			  KeyFile: os.Getenv("CONSUL_TLSKEY"),
-			  InsecureSkipVerify: false,
+		tlsConfigDesc := &consulapi.TLSConfig{
+			Address:            uri.Host,
+			CAFile:             os.Getenv("CONSUL_CACERT"),
+			CertFile:           os.Getenv("CONSUL_CLIENT_CERT"),
+			KeyFile:            os.Getenv("CONSUL_CLIENT_KEY"),
+			InsecureSkipVerify: false,
```

Comment:
I humbly and politely request `InsecureSkipVerify` be an environment variable (default: `false`).
 
Perhaps `os.Getenv("CONSUL_INSECURE")`?  Keeps the same ominous "visual warning" as the words "InsecureSkipVerify" when setting to `true`.